docs/CONTRIBUTING.md

```diff
@@ -82,6 +83,10 @@ done the following things first:
 [la]: LICENSE-APACHE
 [rustfmt]: https://github.com/rust-lang-nursery/rustfmt
 
+Note that if you do not have mdbook already installed you may do so with `cargo install mdbook`. 
+If you find dependency resolution problems when testing mdbook, 
```

Comment:
how about, "If mdbook fails with a multiple crate resolution error"

Reply:
Would "If testing mdbook fails with crate resolution errors," work?

Reply:
it wouldn't always work -- there can be snippets that use `extern crate something`, but the `something` is missing from (dev-?) dependencies and it would give a different kind of crate resolution error